pkg/koordlet/koordlet.go

```diff
@@ -154,6 +157,14 @@ func (d *daemon) Run(stopCh <-chan struct{}) {
 		}
 	}()
 
+	// create be pod cpuset dir
```

Comment:
how about we just read cpuset info from one BE pod/container, so that we don't need to maintain a temp cgroup path

Reply:
done